01_Build_Process/05_Generating_Iso.md

```diff
@@ -65,20 +65,24 @@ That's a lot of flags! Because of how flexible the iso format is, we need to be
 Now we have an iso with our kernel, config and bootloader files on it. We just need to install limine into the boot partitions, we can do this using `limine-deploy` like so:
 
 ```sh
-limine-deploy my_iso.iso
+./limine/limine bios-install $(IMAGE_NAME).iso
```

Comment:
you use `$(IMAGE_NAME)` here but I dont think its defined, seems I previously used `my_iso.iso` as the filename, can we use that for consistency?